.changeset/cloudflare-entrypoint-breaking.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/cloudflare': major
+---
+
+Custom entrypoint API has changed
+
+The `createExports()` function has been replaced with a direct export pattern. If you're using a custom `entryPoint` in your Cloudflare adapter config, update your worker file from:
+
+```ts
+import { createExports } from 'astro/app';
+export const { default } = createExports(manifest);
+```
+
+to:
+
+```ts
+import { handle } from '@astrojs/cloudflare/utils/handler';
+export default { fetch: handle };
```

Comment:
We should show a more concrete example, following the example we currently have in docs